ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: openshift-e2e-test-qe
+  steps:
+  - ref: cucushift-pre
+  - ref: cucushift-e2e
+  - ref: openshift-extended-test
```

Comment:
There is a `cucushift-e2e-report` step, can we consider creating a unified step similar to this to summarize all e2e failures? Therefore we will be able to let all E2E return 0 and signal job success/failures using a final step.

In order to do so, the E2E need to write things to the `SHARED_DIR`, check the `cucushift-e2e-commands.sh` for example.

Reply:
Done.